x-pack/plugins/xpack_main/server/lib/replace_injected_vars.js

```diff
@@ -5,13 +5,17 @@
  */
 
 import { getTelemetryOptIn } from "./get_telemetry_opt_in";
+import { populateUICapabilities } from './populate_ui_capabilities';
 
 export async function replaceInjectedVars(originalInjectedVars, request, server) {
   const xpackInfo = server.plugins.xpack_main.info;
   const withXpackInfo = async () => ({
     ...originalInjectedVars,
     telemetryOptedIn: await getTelemetryOptIn(request),
     xpackInitialInfo: xpackInfo.isAvailable() ? xpackInfo.toJSON() : undefined,
+    uiCapabilities: {
+      ...populateUICapabilities(server.plugins.xpack_main, originalInjectedVars),
```

Comment:
nit: if all that we're using within the `populateUICapabilities` function is the uiCapabilities, I'd recommend passing `originalInjectedVars.uiCapabilities` to `populateUICapabilities`.